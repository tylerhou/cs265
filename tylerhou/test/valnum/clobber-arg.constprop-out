@main {
.__b0:
  a: int = const 1;
  b: int = const 2;
.lbl:
  b: int = const 3;
}
