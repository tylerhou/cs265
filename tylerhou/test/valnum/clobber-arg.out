@main {
.__b0:
  a: int = const 1;
  b: int = const 2;
.lbl:
  b: int = add a b;
}
